content/en/blog/2022/2022-03-15-k8s-triage-bot-helper-ci-job/K8s Triage bot Helper CI Job_ ‘make update’.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: blog
+title: "K8s Triage bot Helper CI Job: ‘*make update*’"
+date: 2022-03-15
+slug: k8s-triage-bot-helper-ci-job
+---
+
+**Authors:** [Subhasmita Swain](https://github.com/SubhasmitaSw), [Davanum Srinivas](https://github.com/dims) 
+
+---
+
+
+If you are contributing to the Kubernetes project and are developing on a Windows PC, it is conceivable that you will encounter certain issues that will cause your patches to master to somehow be halted. This blog is a workaround for a similar issue I encountered when attempting to have my modifications approved and merged into the master repository.
+
+## Why is this needed?
+ 
+While contributing to [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) for some minor documentation changes, the pushed changes needed to be updated with other verified contents of the entire documentation. So, in order for the change to take effect, a single command must be performed to ensure that all tests on the CI pipeline pass. The single command `make update` runs all presubmission verification tests. For some reason on the windows `wsl` environment the tests, specifically the [update-openapi-spec.sh](https://github.com/SubhasmitaSw/kubernetes/blob/master/hack/update-openapi-spec.sh) script, failed (in my case, take a look at the conversation [here](https://github.com/kubernetes/kubernetes/pull/107691)), eventually failing the `pull-kubernetes-verify` tests.
```

Comment:
The link `https://github.com/SubhasmitaSw/kubernetes/blob/master/hack/update-openapi-spec.sh` should be `https://github.com/kubernetes/kubernetes/blob/master/hack/update-openapi-spec.sh`

(it's OK to replace `master` with a specific commit).

Reply:
Change “windows `wsl`” to “WSL”, or write “Windows Subsystem for Linux”